build.gradle

```diff
@@ -1,36 +1,94 @@
 plugins {
-    id "com.jfrog.bintray" version "1.1"
-    id "com.github.jruby-gradle.base" version "0.1.5"
     id "java"
     id "checkstyle"
+    id "maven-publish"
+//    id "signing"
```

Comment:
Let's have `"signing"`.

Reply:
Enable `signing`

There is no configuration about Maven Central yet.
So I don't enable the following parts yet.

```java
// TODO: Enable those settings after append maven configuration.
//signing {
//    sign publishing.publications.maven
//}
```